specification/search_application/render_query/SearchApplicationsRenderQueryResponse.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export class Response {
+  body: {}
```

Comment:
Q: Do we somewhere track these cases where we only defined a request, but not a proper response?

Doing it like this will suppress the compiler warning about missing request/response.

Reply:
Good point. Does this mean it's preferable to omit the response file entirely when we don't know the contents of the body?

Reply:
> Doing it like this will suppress the compiler warning about missing request/response.

I'm also surprised that `make validate` passed, which is why I did not realize that the body was missing.

> Good point. Does this mean it's preferable to omit the response file entirely when we don't know the contents of the body?

Ideally, yes, but it breaks the OpenAPI transformation right now.